spec/index.bs

```diff
@@ -671,15 +671,29 @@ initially empty.
 
     1. Set |localName| to |tagName| in [=ASCII lowercase=].
     1. If |elementNs| is an empty string, set |elementNs| to [=HTML namespace=].
+    1. If |property| starts with "on", then return {{TrustedScript}}.
```

Comment:
Had to look it up, so perhaps worth linking from a note to https://html.spec.whatwg.org/#event-handler-name?

E.g.
```
 <p class="note" role="note"><span class="marker">Note:</span> ...
```

Reply:
Ah good point